testsuite/features/support/api_test.rb

```diff
@@ -83,11 +103,17 @@ def date_now
 
 # Derived class for an HTTP test
 class ApiTestHttp < ApiTest
+  ##
+  # It creates a new instance of the HttpClient class.
+  #
+  # Args:
+  #   host: The hostname of the server.
   def initialize(host)
     super
     @connection = HttpClient.new(host)
   end
 
+  ##
   # during HTTP tests, dates are strings
```

Comment:
I know this isn't what you changed but this doesn't really explain what the function does. does it convert string dates into Date objects? if so, that should be a bit more clear in the description

Reply:
@Bischoff could you maybe add a suggested change comment here with a better explanation?
I remember you wrote this part.

Reply:
Adding documentation for every function will be a huge task, and i'm not sure of the benefit, especially that autogenerated documentation is usually not really helping. Maybe we should just keep this rubocop error masked?

To answer your question: XML-RPC uses Date class for dates, while HTTP RPC uses simple strings for dates. This function provides a string containing a date that can be swallowed by HTTP RPC. *I accept my description was not the best...

Reply:
I added your description. Resolving.

Reply:
it does? it seems to return a boolean though? :thinking: this might just be my noob self at ruby not understanding, sorry :sweat: but I promise to write a really good description as a suggestion once I do understand